app/models/reviewable.rb

```diff
@@ -66,14 +66,11 @@ def self.default_visible
   end
 
   def self.valid_type?(type)
-    return false if Reviewable.types.exclude?(type)
-    type.constantize <= Reviewable
```

Comment:
This check is now done in `Plugin::Instance#register_reviewable_type`, so a plugin can’t add a new type that wouldn’t inherit from `Reviewable`.